recipes/steem/meta.yaml

```diff
@@ -80,9 +80,11 @@ test:
     - steembase.storage
     - steembase.transactions
     - steembase.types
+
   commands:
     - conda inspect linkages -p $PREFIX steem  # [not win]
     - conda inspect objects -p $PREFIX steem   # [osx]
+    - python steempy --help
```

Comment:
Does this require a call to python? https://github.com/steemit/steem-python#cli-tools-bundled